data/Cowling_1987/data.csv

```diff
@@ -0,0 +1,5 @@
+﻿Species,flowering_time,resprouting_capacity,bud_bank_location,serotiny,germination_%,seed_dry_mass_mg,seed_length_min_mm,seed_length_max_mm,seed_width_min_mm,seed_width_max_mm,reproductive_maturity_min,reproductive_maturity_max,reproductive_maturity_mean
```

Comment:
I've just swapped it - that is a really interesting catch. Because that is how we defined it, because we decided very few people would make measurements based on the "proper" orientation of the seed.